### PR TITLE
[FIX] l10n_ar_ux: uninstall hooks for monkey patches

### PR DESCRIPTION
**Pull request by cav-adhoc**

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina/pull/920).

File: l10n_ar_ux/hooks.py [hunk 1]
```diff
@@ -5,6 +5,8 @@
 from odoo import api, SUPERUSER_ID
 import logging
 _logger = logging.getLogger(__name__)
+from odoo.addons.l10n_ar.models.account_move import AccountMove as AccountMoveAr
+from odoo.addons.l10n_ar.models.account_fiscal_position import AccountFiscalPosition
```
Comment (maq-adhoc) [inline l10n_ar_ux/hooks.py:9]: from odoo.addons.l10n_ar.models.account_fiscal_position.AccountFiscalPosition import _get_fiscal_position as _get_fiscal_position_original

File: l10n_ar_ux/hooks.py [hunk 2]
```diff
@@ -29,3 +31,16 @@ def post_init_hook(cr, registry):
...
+    def _inverse_l10n_latam_document_number(self):
+        AccountMoveAr._inverse_l10n_latam_document_number(self)
+
+    AccountMoveAr._inverse_l10n_latam_document_number = _inverse_l10n_latam_document_number
+
+    @api.model
+    def _get_fiscal_position(self, partner, delivery=None):
+        return super(AccountFiscalPosition, self)._get_fiscal_position(partner, delivery=delivery)
+
+    AccountFiscalPosition._get_fiscal_position = _get_fiscal_position
```
Comment (maq-adhoc) [inline l10n_ar_ux/hooks.py:46]: AccountFiscalPosition._get_fiscal_position = _get_fiscal_position_original

**Review by maq-adhoc — CHANGES_REQUESTED** [review]
Vemos si funciona

**Comment by cav-adhoc** [comment]
sigue en https://github.com/ingadhoc/odoo-argentina/pull/952

**Pull request closed without merging**